conda_smithy/templates/build_and_run.sh.tmpl

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
Instead of having a new file, can we re-use `build_steps.sh` ?

You can change the existing file like

```diff
diff --git a/conda_smithy/templates/build_steps.sh.tmpl b/conda_smithy/templates/build_steps.sh.tmpl
index e75b5bb..9f51414 100644
--- a/conda_smithy/templates/build_steps.sh.tmpl
+++ b/conda_smithy/templates/build_steps.sh.tmpl
@@ -7,15 +7,15 @@
 
 set -xeuo pipefail
 export PYTHONUNBUFFERED=1
-export FEEDSTOCK_ROOT=/home/conda/feedstock_root
-export RECIPE_ROOT=/home/conda/recipe_root
-export CI_SUPPORT=/home/conda/feedstock_root/.ci_support
+export FEEDSTOCK_ROOT=${FEEDSTOCK_ROOT:-/home/conda/feedstock_root}
+export RECIPE_ROOT=${RECIPE_ROOT:-/home/conda/recipe_root}
+export CI_SUPPORT=${FEEDSTOCK_ROOT}/.ci_support
 export CONFIG_FILE="${CI_SUPPORT}/${CONFIG}.yaml"
 
 cat >~/.condarc <<CONDARC
 
 conda-build:
- root-dir: /home/conda/feedstock_root/build_artifacts
+ root-dir: ${FEEDSTOCK_ROOT}/build_artifacts
 
 CONDARC
 
@@ -39,4 +39,4 @@ if [[ "${UPLOAD_PACKAGES}" != "False" ]]; then
     upload_package "${FEEDSTOCK_ROOT}" "${RECIPE_ROOT}" "${CONFIG_FILE}"
 fi
 
-touch "/home/conda/feedstock_root/build_artifacts/conda-forge-build-done-${CONFIG}"
+touch "${FEEDSTOCK_ROOT}/build_artifacts/conda-forge-build-done-${CONFIG}"
```

And in drone, before calling this file `FEEDSTOCK_ROOT` and `RECIPE_ROOT` is set and the conda environment is activated.

Reply:
Good call, I wanted to reuse that script but couldn't think of a good way.  Let me see if I can get that working.

Reply:
Updated.